README.md

```diff
@@ -240,6 +238,63 @@ export default Ember.Route.extend({
 });
 ```
 
+`ember-ajax` comes with helpers for matching response errors to their
+respective `ember-ajax` error type. Each of the errors listed above has a
+corresponding `is*` method (e.g., `isBadRequestError`).
+
+Use of these methods is **strongly encouraged** to help eliminate the need for
+boilerplate error detection code. Each of the built-in helpers may be 
+customized in your `AjaxService` if you need extra logic for matching errors.
+
+```js
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  ajax: Ember.inject.service(),
+  model() {
+    const ajax = this.get('ajax');
+    const {isNotFoundError} = ajax;
+
+    return ajax.request('/user/doesnotexist')
+      .catch(function(error) {
+        if (ajax.isNotFoundError(error)) {
```

Comment:
I'm not sure that this is actually how these were intended to be used...

In my mind, at leas, the `isNotFoundError` methods were private on the service so that they could be overwritten as needed for usage within the `ajax` service itself.  If you want to do error detection within the application, you import the method from `ember-cli-ajax/errors` and use it directly.